cmd/user-agent/src/pages/DIDManagement.vue

```diff
@@ -0,0 +1,139 @@
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+<template>
+    <div class="content">
+        <div class="md-layout">
+            <div
+                    class="md-layout-item md-medium-size-100 md-xsmall-size-100 md-size-100"
+            >
+                <md-card class="md-card-plain">
+                    <md-card-header data-background-color="green">
+                        <h4 class="title">Create your DID</h4>
+                        <p class="category"> Create button will create and save the did in DID store</p>
+                    </md-card-header>
+                    <md-card-content style="background-color: white;">
+                        <md-field>
+                        </md-field>
+                        <select id="selectVC" v-model="selectType" style="color: grey; width: 200px; height: 35px;">
+                            <option value="" disabled="disabled">Select Key Type</option>
+                            <option value="Ed25519">Ed25519VerificationKey2018</option>
+                            <option value="P256KeyType">P256KeyType</option>
+                        </select>
+                        <md-field style="margin-top: -15px">
+                        </md-field>
+                        <div class="md-layout-item md-size-100">
+                            <md-field maxlength="5">
+                                <label class="md-helper-text">Type DID friendly name here</label>
+                                <md-input v-model="friendlyName" id="friendlyName" required></md-input>
+                            </md-field>
+                        </div>
+                        <md-button class="md-button md-info md-square md-theme-default md-large-size-100 md-size-100"
+                                    id='createDIDBtn' v-on:click="createDID">Create and Save DID
+                        </md-button>
+                        <div v-if="errors.length">
+                            <b>Please correct the following error(s):</b>
+                            <ul>
+                                <li v-for="error in errors" :key="error">{{ error }}</li>
+                            </ul>
+                        </div>
+                        <md-field>
+                        </md-field>
+                        <md-field>
+                            <md-textarea v-model="didDocTextArea" readonly rows="20" cols="100">
+                            </md-textarea>
+                        </md-field>
+
+
+                    </md-card-content>
+                </md-card>
+            </div>
+        </div>
+    </div>
+</template>
+<script>
+    export default {
+        beforeCreate: async function () {
+            window.$aries = await this.$arieslib
+            window.$trustblocAgent = this.$trustblocAgent
+            window.$trustblocStartupOpts = await this.$trustblocStartupOpts
+
+        },
+        methods: {
+            createDID: async function () {
```

Comment:
I see this using the trustblocagent... which probably only creates trustbloc DIDs...

Is there a plan to create other kinds of DIDs (eg peer)?

Reply:
I am not aware of that, as of now this was the existing implementation. I refactored UI component of that. I know we should be able to support P256 crypto for trustbloc did. Not sure about the priority for supporting other dids.  @fqutishat  would you know this. ?

Reply:
We should create an issue to support additional methods.

Reply:
#111 is created